site/content/extend/plugins/community-plugin-marketplace.md

```diff
@@ -0,0 +1,90 @@
+2. The plugin works as expected with the latest ESR version of Mattermost. This must not be checked if `min_server_version` is higher than the latest ESR version.
+
+Given that plugins are currently in Beta and rapid development is still happening, breaking changes especially for the webapp can happen. It is expected from plugin authors to keep up with these changes and fix bugs that may occur. Breaking changes will be mentioned in the "Important Upgrades Notes" section of the [Mattermost Changelog](https://docs.mattermost.com/administration/changelog.html).
+
+### Requirements for updating a community plugin to the Marketplace
+On every update of a community plugin must still fulfill the following checklist to be added to the Marketplace. This is checked by the four reviewers in the same way as when the plugin was added. The code review and security review should be performed against the `diff` of the last version in the Marketplace and the new version.
+
+The release also has to follow [semver](https://semver.org/). This specifically means for plugins:
+
+* If the plugin exposes an API for inter plugin communication, breaking changes to the API require a major version bump.
+* If an update requires manual migration actions from the administrator, a major version bump is required.
+
+This is checked in dev review.
+
+The new release must not change the plugin id defined in the manifest as this would require a reconfiguration of the plugin by a system admin.
+
+### Process for adding community plugin to the Marketplace
+There will be an owner for community plugins, that will guide you through the review process. Currently this is [hanzei](https://github.com/hanzei).
+
+1. You open an issue on [The Plugin Marketplace Repository](https://github.com/hanzei/plugin-test/issues/new/choose) using a pre defined template for new plugins. The template contains the checklist above, so you can check the items. Please also points out which commit should be used for the review. You may cut an release candidate (RC) and for it the reviews.
+2. The reviews are requested by the owner. This is done by cloning the following JIRA ticket for every reviewer, assigning it and mentioning the ticket on GitHub. The reviewers point out found issues on the bug tracker of the community plugin. Once all issues are resolved, they comment on the issue stating the positive review and mark the JIRA ticket as resolved.
+    - [PM](https://mattermost.atlassian.net/browse/MM-22224)
+    - [Dev](https://mattermost.atlassian.net/browse/MM-22221)
+    - [Security](https://mattermost.atlassian.net/browse/MM-22225)
+    - [QA](https://mattermost.atlassian.net/browse/MM-22223)
+3. Once all items are checked and reviews made, the community repository is forked under the Mattermost GitHub organization as a private repository. This allows using the existing build tools for releasing new plugin versions. The fork is maintained by the owner.
```

Comment:
If the owner is non a core contributor, how do they maintain the private fork? Should this fork occur before the review is done instead?

Reply:
I didn't counted for a case where the owner is not a core contributor. 

To be clear: This is referring to the owner in https://github.com/mattermost/mattermost-developer-documentation/pull/524/files/0f1aa012672b64c31e5d15a2ee74c655085b9ba7#diff-ee526eb501deade82cd69d56e8654e3aR54. In most cases I expect myself to be the owner.

Reply:
Ah, that makes sense. I didn't interpret `owner` earlier in the document as a "defined term", but just something used in passing, so the context here wasn't clear. But going further, I'm not convinced that an `owner` even in this context is permanent -- we should be assigning an owner each time the plugin is updated or if Mattermost takes some action.

I think it's important to distinguish the audiences here -- the plugin authors won't see these internal details, and might interpret them as actions they have to take. Anything we can do to disambiguate would be excellent!

Reply:
Thanks for the feedback. I agree that a owner doesn't have to be permanent. 

I've updated the document to make more clear that owner is a defined term by writing it in italic. I hope this term makes it clear, that these are actions taken by Mattermost and not the community member.